addons/edi/tests/common.py

```diff
@@ -46,6 +46,11 @@ class EdiCase(common.SavepointCase):
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
+        patch_cr = patch.object(
```

Comment:
Please add a comment about the significance of patching (commit in the middle of savepoint etc.) so that will help you two weeks later :smiley:

Reply:
I forgot already tbh, your memory is quite good :)